HLTrigger/HLTanalyzers/src/HLTInfo.cc

```diff
@@ -557,7 +557,7 @@ void HLTInfo::analyze(const edm::Handle<edm::TriggerResults>                 & h
       // ...Fill the corresponding accepts in branch-variables
       l1flag[iBit] = gtDecisionWord[iBit];
 
-      std::string l1triggername= std::string (algoBitToName[iBit]);
+      std::string l1triggername= std::string (static_cast<const char *>(algoBitToName[iBit]));
```

Comment:
sigh... I'm actually sorry for having this kind of code under HLTrigger/, even though it's not used in any production workflow (and possibly not used any more at all, since it doesn't look like it was ever ported to the L! trigger used in Run 2).
OK for the minimal fix, I'll add it to my list of things to be cleaned up (or dropped).